signal_analog/filters.py

```diff
@@ -0,0 +1,73 @@
+import signal_analog.util as util
+
+
+class FilterVariable(object):
```

Comment:
Can we add docstrings here for `FilterVariable`? We've just been pulling them straight from the sfxdocs for now

Reply:
Thanks for pointing that out. They have been added